deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,120 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+from deepchem.utils.molecule_feature_utils import one_hot_encode
+import numpy as np
+from rdkit import Chem
+from sklearn.metrics import pairwise_distances
+
+
+class MATFeaturizer(MolecularFeaturizer):
+  """
+  This class is a featurizer for the Molecule Attention Transformer [1]_.
+  The featurizer accepts an RDKit Molecule, and 2 booleans (add_dummy_node and one_hot_formal_charge) as arguments.
+  The returned value is a tuple which consists of molecular graph descriptions:
```

Comment:
To see how to add examples, see https://numpydoc.readthedocs.io/en/latest/format.html#examples. You'll want to do something like

```
Examples
--------
>>> feat = MATFeaturizer(...)
>>> out = feat.featurize("CCC")
```

Reply:
Done! Please have a look.